buildSrc/build.gradle.kts

```diff
@@ -30,6 +30,7 @@ dependencies {
     testImplementation("com.pinterest.ktlint:ktlint-core:$ktlintVersion")
     testImplementation("com.pinterest.ktlint:ktlint-test:$ktlintVersion")
 
+    implementation("org.jacoco:org.jacoco.core:0.8.6")
```

Comment:
nit: put version in gradle properties file

Reply:
Pulls version from the Gradle plugin since we use the default